java-client/barrage/src/main/java/io/deephaven/client/impl/BarrageSnapshot.java

```diff
@@ -38,28 +38,12 @@ BarrageSnapshot snapshot(TableSpec tableSpec, BarrageSnapshotOptions options)
         BarrageSnapshot snapshot(TableHandle tableHandle, BarrageSnapshotOptions options);
     }
 
-    /**
-     * This call will return false until all rows for the result table are available or the snapshot failed.
-     *
-     * @return true when all rows for the result table are available or the snapshot failed, false otherwise
-     */
-    boolean isCompleted();
-
     /**
      * Request a full snapshot of the data and populate a {@link Table} with the data that is received.
      *
      * @return the {@code Table}
      */
-    Table entireTable() throws InterruptedException;
-
-    /**
-     * Request a full snapshot of the data and populate a {@link Table} with the data that is received.
-     *
-     * @param blockUntilComplete Whether to block execution until all rows for the subscribed table are available
-     *
-     * @return the {@code Table}
-     */
-    Table entireTable(boolean blockUntilComplete) throws InterruptedException;
+    Future<Table> entireTable() throws InterruptedException;
```

Comment:
We don't throw an `InterruptedException` anymore. We have a `Future` for that. Applies to all these interfaces.